build.xml

```diff
@@ -1385,6 +1385,28 @@
     </antcall>
   </target>
 
+  <!--
+    ant test-iteration -Dtest.name=org.apache.cassandra.service.StorageServiceServerTest -Dtest.iters=5
+    ant test-iteration -Dtest.name=org.apache.cassandra.cache.AutoSavingCacheTest -Dtest.iters=1
+    ant test-iteration -Dtest.name=org.apache.cassandra.cache.AutoSavingCacheTest -Dtest.methods=testSerializeAndLoadKeyCache -Dtest.iters=2
+    Notice: it cannot work since JDK15
```

Comment:
I think we have already another approach in place.

https://github.com/apache/cassandra/blob/trunk/.build/README.md?plain=1#L122-L131